### ISSUE TITLE
make shortcuts configurable 

### ISSUE THREAD

**Issue by KryDos** [KryDos/todoist-linux#14]
I'm currently think about `~/.todoist.linux.json` file that will store all different configuration including shortcuts. 

So users will be able to edit this file freely.

**Comment by KryDos** [comment]
cc: @roock

**Comment by roock** [comment]
Did you have a look at https://github.com/kamhix/todoist-linux lately? He did release a new version in late April which supports configurable global shortcuts - although his application has an annoying bug and global shortcuts are not working perfectly, so I'm currently using your app :smile:

**Comment by KryDos** [comment]
yeah, I've used his app initially but I've got some issues on my Ubuntu that didn't allow me to use it (there was some issues with tray if I'm not mistaken). I thought Kamhix isn't supporting the app anymore but I see there are new commits which is great. 

I don't know if I can borrow his code related to shortcuts because I don't want to do this without his permission and because my app is using electron while his is written with `nw.js`. 

Thank you for the link though, I'll take a look how shortcuts are implemented in Kamhix app and if it's something nice I'll ask him if I can take it 🤓

**Comment by roock** [comment]
Thx. works perfectly for me :) :+1:
